src/API/timeseries/timeseries.controller.js

```diff
@@ -0,0 +1,66 @@
+const PERIOD_TO_INTERVAL = {
+    'day': 60,
+    'week': 60,
+    'month': 60*24,
+    'year': 60*24
+};
+
+import APIError from '../helpers/APIError';
+import httpStatus from 'http-status';
+import moment from 'moment';
+
+function query(req, res, next) {
+
+    if (!req.query.period || ! PERIOD_TO_INTERVAL.hasOwnProperty(req.query.period)) {
+        const err = new APIError('Period is not provided to wrong', httpStatus.BAD_REQUEST);
+        return next(err);
+    }
+
+    if (!req.query.sensor) {
+        const err = new APIError('Sensor is not provided', httpStatus.BAD_REQUEST);
+        return next(err);
+    }
+
+    const sensor = req.query.sensor;
```

Comment:
the const and let declarations is block scoped, therefor decided to keep it like it is